src/Package/Impl/ProjectSystem/Commands/DisabledProjectCommand.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Immutable;
+using Microsoft.VisualStudio.ProjectSystem;
+#if VS14
+using Microsoft.VisualStudio.ProjectSystem.Designers;
+#endif
+
+namespace Microsoft.VisualStudio.R.Package.ProjectSystem.Commands {
+    internal abstract class DisabledProjectCommand : ICommandGroupHandler {
```

Comment:
Can't quite figure out what the name for this means. Looking at the code, it's more like `ProjectNodeSelectionCommand`?